### ISSUE TITLE
Regarding offline usage

### ISSUE THREAD

**Issue by devarajarun** [SebastianStaab/coracle#2]
Is it possible to use this tool offline? If yes, are there any tutorials or sample commands available for use?

**Comment by SebastianStaab** [comment]
There are tutorials on the data preparation available at https://micportal.org/coracle/
Once your data is prepared you can simply call coralce like a function, e.g.:
from coracle import coracle
result = coracle(x, y)
#where x and y are your respective datasets

**Comment by devarajarun** [comment]
should I use the file coracle.py ?

**Comment by SebastianStaab** [comment]
Exactly, you'll also need the _cfs.py_ file. Put them in the same directory as the new file (where you run it) and depending on your interpreter you will need to execute _cfs.py_ first, then _coracle.py_ so that the **import** command

**Comment by devarajarun** [comment]
Before that should I need to install the dependencies like pandas, sklearn etc.

**Comment by SebastianStaab** [comment]
Yes. Sometimes they are already included in your python distribution.

**Comment by devarajarun** [comment]
I am getting this error with 'np.float' in asgl.py

`AttributeError:` module 'numpy' has no attribute 'np.float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    `https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations`

again I tried with 'float' in asgl.py and I still get the same error. please help me to solve this.

**Comment by devarajarun** [comment]
It seems to be working now.  I am getting 2 %..3 %..5 %..7 %..8 %....
I am just curious if it makes use of all available cores and memory or if it needs to be added manually. Just wanna speed up the process.

**Comment by devarajarun** [comment]
But thanks anyhow. I received the result.
For your reference, this is the code I used. 

`# I used conda env for version compatibility
conda create -n coracle python==3.9
conda activate coracle
pip install numpy==1.23.5 pandas scikit-learn cfs asgl
python cfs.py
python coracle.py
python
>>> from coracle import coracle
>>> import pandas as pd
>>> x = pd.read_csv('ASV.csv', index_col=0)
>>> y = pd.read_csv('Temp.csv', index_col=0)
>>> result = coracle(x, y)
>>> result.to_csv('coracle_out.csv', index=True)`
